pkg/apis/serving/v1alpha1/kfservice_status.go

```diff
@@ -14,41 +14,82 @@ limitations under the License.
 package v1alpha1
 
 import (
-	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	"github.com/knative/pkg/apis"
 	knservingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"k8s.io/api/core/v1"
 )
 
 // ConditionType represents a Service condition value
 const (
-	// ServiceConditionRoutesReady is set when the service's underlying
+	// KFServiceConditionRoutesReady is set when the service's underlying
```

Comment:
I worry a little about exposing the Knative "Route" primitive in our interface. I think using the terminology "Route" is fine though. Do you mind rewording the comment to not expose the detail? Maybe something like, "once network configuration has completed"

Reply:
removed in comments.